clean_enoa/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""An amazing sample package!"""
```

Comment:
¿Por favor podríamos mejorar este comentario? No propongo una alternativa porque apenas me estoy familiarizando.

Reply:
Listo, agregué una pequeña descripción:
"This package make request about unemployment in Mexico to INEGI"

Reply:
"This package makes requests about unemployment in Mexico to INEGI"